Documentation/contributing/testing/unit.rst

```diff
@@ -119,125 +105,3 @@ run this in a terminal next to your editor:
     $ watchtest pkg/policy
 
 This shell script depends on the ``inotify-tools`` package on Linux.
-
-.. _migrate-gocheck:
```

Comment:
We're still referencing this section in `.golangci.yaml` for the `gomodguard` linter:

https://github.com/cilium/cilium/blob/5f0e3e21ae40b7f8537e4949652aa99b98b264ab/.golangci.yaml#L69-L73

Probably better to link to https://github.com/cilium/cilium/issues/28596 instead now.

On the same note, it might make sense to include `github.com/cilium/checkmate` in that list as part of this PR as well so we don't accidentally reintroduce it.

Reply:
good point ✅